addons/mass_mailing_crm/static/src/crm_lead_gen_element.js

```diff
@@ -0,0 +1,22 @@
+import { patch } from "@web/core/utils/patch";
+import {
+    LeadGenerationDropdown, MODULE_STATUS
+} from "@crm_iap_mine/components/lead_generation_dropdown/lead_generation_dropdown";
+
+patch(LeadGenerationDropdown.prototype, {
+    setup() {
+        super.setup();
+        const mailingElement = this.state.dropdownContentElements.find(element => element.moduleXmlId === 'base.module_mass_mailing');
```

Comment:
🧦 The line is a bit long here so

```suggestion
        const mailingElement = this.state.dropdownContentElements.find(
            (element) => element.moduleXmlId === 'base.module_mass_mailing'
        );
```